### DISCUSSION TITLE
Add Functionality to Display Execution Date and Time for Each Cell

### DISCUSSION THREAD

**Discussion by QuanHongLiu** [microsoft/vscode-jupyter#16081] [Questions and Answers]
Hello, thank you for the excellent work on the VS Code Jupyter extension! I would like to propose a feature request to automatically display the execution date and time in the output for each cell in Jupyter Notebooks.

Currently, the built-in Notebook Output in VS Code only shows the execution duration of each cell, but it does not display the specific date and time. This would be incredibly useful for tracking data analysis and experimental results, especially when dealing with long-running computations or when time-stamping logs is necessary.

Desired Functionality:  
- At the completion of each cell execution, the output should include:
  - Start Time (e.g., 2024-09-28 15:45:20)
  - End Time (e.g., 2024-09-28 15:45:22)
  - Execution Time*(e.g., 2.00 seconds)

This would help users better understand and document the execution process of their code.

Additional Suggestions:  
If possible, it would be great to provide a configuration option that allows users to choose whether or not to display this information in the output.

Thank you for considering this request! I look forward to your response.

**Comment by barnysanchezciq** [comment]
I wished this was included ASAP, but I couldn´t agree more on its usefulness. Execution time is already provided. I don´t know if this is configurable at the time to make it report in `ms` as opposed to `s` as in mini tests the projected drops of time from for example `1000 ms` to `150 ms` is indicative of a massive efficiency gain and can't deduce that from seconds as both would report as `0.0s` (see attachment). 

The desired output in my mind would be something like: `1,800 ms | 2024-09-28 15:45:20 | 2024-09-28 15:45:22` 

Bonus if the execution time would be configurable (maybe settings) to display in minutes, seconds, and milliseconds so a more extended execution example could display like: `8m, 2s, 150 ms | 2024-09-28 15:45:20 | 2024-09-28 15:53:22`

Some rounding up or down, expected! 

![image](https://github.com/user-attachments/assets/b818af67-5453-4469-997d-a7e43ddabd83)
